misc/start_faux

```diff
@@ -25,11 +25,44 @@ else
     exit 1
 fi
 
+set_mac_addrs () {
+  echo Setting alternate mac $1 on $intf_name
+  ip link set addr $1 dev $intf_name
+}
+
+set_passwd () {
+  echo $1 | chpasswd
+}
+
 # Setup change-of-mac address for macoui testing.
 if [ -n "${options[macoui]}" ]; then
     new_mac=3c:5a:b4:1e:8f:0a
-    echo Setting alternate mac $new_mac on $intf_name
-    ip link set addr $new_mac dev $intf_name
+    set_mac_addrs $new_mac
+fi
+
+if [ -n "${options[passwordpass]}" ]; then
+    set_passwd 'root:pass'
+    htpasswd -b -c /etc/nginx/.htpasswd admin fail
+    /etc/init.d/openbsd-inetd restart
+    /etc/init.d/xinetd restart
+    /usr/sbin/sshd -D &
+    service ssh start
+
+fi
+
+if [ -n "${options[passwordfail]}" ]; then
+    new_mac=3c:5a:b4:1e:8f:0b
+    set_mac_addrs $new_mac
+    set_passwd 'root:redstone'
```

Comment:
don't encode redstone here -- this will be OSS and not owned... find a generic name to use instead (here and elsewhere)

Reply:
fixed